src/mmw/js/src/modeling/gwlfe/entry/calcs.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+
+var _ = require('lodash'),
+    turfArea = require('turf-area'),
+    App = require('../../../app');
+
+/**
+ * This module exports a set of objects for calculating outputs of, and auto-
+ * estimatates for, a manual entry field for GWLF-E.
+ *
+ * Each calculator object has two methods: `toOutput` which given a user
+ * value converts it to the expected GMS format, and `getAutoValue` that
+ * gets a user-friendly value from the GMS dataModel given a field name.
+ *
+ * Each manual entry field will be configured with one of these calculators.
+ */
+
+module.exports = {
+    // Takes the userValue and uses it directly in the output
+    Direct: {
+        toOutput: function(userValue) {
+            return userValue;
+        },
+        getAutoValue: function(fieldName, dataModel) {
+            return dataModel[fieldName];
+        },
+    },
+
+    // Takes the userValue and makes an array with the value repeated 12 times
+    Array12: {
+        toOutput: function(userValue) {
+            return [userValue, userValue, userValue,
+                    userValue, userValue, userValue,
+                    userValue, userValue, userValue,
+                    userValue, userValue, userValue];
+        },
+        getAutoValue: function(fieldName, dataModel) {
+            return dataModel[fieldName][0];
+        },
+    },
+
+    // Takes the userValue and divides into an array of 12 values evenly
+    EqualMonths: {
+        toOutput: function(userValue) {
+            var monthValue = userValue / 12;
+            return [monthValue, monthValue, monthValue,
+                    monthValue, monthValue, monthValue,
```

Comment:
I believe this could be `_.fill(12, (userValue / 12))` (and the `Array12` implementation could be `_.fill(12, userValue)`. 

`_.fill` is a little more succinct, but it may be that the less succinct version's easier to comprehend at a glance.